src/assets.ts

```diff
@@ -156,27 +156,22 @@ export class AssetGenerator {
         return ProgramLaunchType.Console;
     }
 
-    private computeProgramPath() {
+    private computeProgramPath(): string {
         if (!this.startupProject) {
             throw new Error("Startup project not set");
         }
 
-        const startupProjectDir = path.dirname(this.startupProject.Path);
-        const relativeProjectDir = path.join('${workspaceFolder}', path.relative(this.workspaceFolder.uri.fsPath, startupProjectDir));
-        const configurationName = 'Debug';
-        const targetFramework = protocol.findNetCoreTargetFramework(this.startupProject);
-        const result = path.join(relativeProjectDir, `bin/${configurationName}/${targetFramework.ShortName}/${this.startupProject.AssemblyName}.dll`);
-        return result;
+        const relativeTargetPath = path.relative(this.workspaceFolder.uri.fsPath, this.startupProject.TargetPath);
```

Comment:
>const relativeTargetPath [](http://example.com/codeflow?start=8&length=24)

Feels like this should really do something like:

```
let targetPath = this.startupProject.TargetPath;
let workspaceRoot = this.workspaceFolder.uri.fsPath;
// TODO: normalize both paths to have the correct directory separator character (forward or backslash), and ensure that workspaceRoot ends in a directory separator
if (targetPath.startsWith(workspaceRoot))
{
   return path.join('${workspaceFolder}', targetPath.Substring(workspaceRoot.Length));
}
else
{
    return targetPath;
}
```

Otherwise I have no idea what this would do if the user modified their output directory to point at a different drive.